src/nimlsppkg/suggestlib.nim

```diff
@@ -9,86 +9,64 @@ export initNimSuggest
 
 proc stopNimSuggest*(nimsuggest: NimSuggest): int = 42
 
-proc `$`*(suggestion: Suggest): string =
-  let sep = ", "
-  result = "(section: " & $suggestion.section
-  result.add sep
-  result.add "symKind: " & $suggestion.symkind.TSymKind
-  result.add sep
-  result.add "qualifiedPath: " & suggestion.qualifiedPath.join(".")
-  result.add sep
-  result.add "forth: " & suggestion.forth
-  result.add sep
-  result.add "filePath: " & suggestion.filePath
-  result.add sep
-  result.add "line: " & $suggestion.line
-  result.add sep
-  result.add "column: " & $suggestion.column
-  result.add sep
-  result.add "doc: " & $suggestion.doc
-  result.add sep
-  result.add "quality: " & $suggestion.quality
-  result.add sep
-  result.add "line: " & $suggestion.line
-  result.add sep
-  result.add "prefix: " & $suggestion.prefix
-  result.add ")"
+proc `$`*(suggest: Suggest): string =
+  fmt"""(section: {suggest.section}, symKind: {suggest.symkind.TSymKind}, qualifiedPath: {suggest.qualifiedPath.join(".")}, forth: {suggest.forth}, filePath: {suggest.filePath}, line: {suggest.line}, column: {suggest.column}, doc: {suggest.doc}, quality: {suggest.quality}, prefix: {suggest.prefix})"""
```

Comment:
In my mind this super long line isn't particularly clean either. I try to stick to below 80 character wide, or at lest not more than 128 characters.